deposit_contract.sol

```diff
@@ -114,8 +112,8 @@ contract DepositContract is IDepositContract, ERC165 {
 
         // Check deposit amount
         require(msg.value >= 1 ether, "DepositContract: deposit value too low");
-        require(msg.value % GWEI == 0, "DepositContract: deposit value not multiple of gwei");
-        uint deposit_amount = msg.value / GWEI;
+        require(msg.value % 1 gwei == 0, "DepositContract: deposit value not multiple of gwei");
```

Comment:
`gwei` became a supported denomination in 0.6.11, no need for our constant.

Reply:
Nice!